src/components/course/data/hooks.jsx

```diff
@@ -226,12 +226,14 @@ export function useCoursePacingType(courseRun) {
  * @returns {Object} { activeCourseRun, userSubsidyApplicableToCourse }
  */
 export const useCoursePriceForUserSubsidy = ({
-  activeCourseRun, userSubsidyApplicableToCourse, courseEntitlements,
+  activeCourseRun, userSubsidyApplicableToCourse, courseEntitlements, canEnrollListPrice,
 }) => {
   const currency = CURRENCY_USD;
   const coursePrice = useMemo(
     () => {
-      const listPrice = activeCourseRun?.firstEnrollablePaidSeatPrice || getEntitlementPrice(courseEntitlements);
+      const listPrice = canEnrollListPrice
```

Comment:
[curious] Given `canEnrollListPrice` now takes into account `activeCourseRun?.firstEnrollablePaidSeatPrice` or `getEntitlementPrice(courseEntitlements);`, should we rely solely on `canEnrollListPrice` at this point?